javascript/ql/src/experimental/Security/CWE-94/examples/ServerSideTemplateInjection.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express')
+var bodyParser = require('body-parser');
+const app = express()
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
```suggestion
const app = express()
```

I think you can simplify the example a bit by skipping the `bodyParser`.   
I don't think `bodyParser` is needed for the example to work? 
(Feel free to reject this suggestion if `bodyParser` is needed for the example).

Reply:
bodyParser is required for example code, but doesnt required for successful exploitation.